src/Features/Core/Portable/Snippets/SnippetProviders/ISnippetProvider.cs

```diff
@@ -12,12 +12,17 @@ internal interface ISnippetProvider
         /// <summary>
         /// What we use to identify each SnippetProvider on the completion list
         /// </summary>
-        string SnippetIdentifier { get; }
+        string Shortcut { get; }
```

Comment:
maybe DisplayText? Not sure what Shortcut means,

Reply:
I was matching what the old snippets used to identify as. It's a shortcut bc it's the shorthand word that brings up the snippet. I can change to display text though